src/features/dashboard/sandboxes/list/table-config.tsx

```diff
@@ -119,6 +119,25 @@ export const resourceRangeFilter: FilterFn<SandboxWithMetrics> = (
   return true
 }
 
+export const templateFilter: FilterFn<SandboxWithMetrics> = (
+  row,
+  _,
+  value: string[]
+) => {
+  if (!value || value.length === 0) return true
+
+  const { alias, templateID } = row.original
+
+  // check if any filter value matches either alias or templateID
+  return value.some((filterValue) => {
+    return (
+      (alias && alias.toLowerCase().includes(filterValue.toLowerCase())) ||
+      (templateID &&
+        templateID.toLowerCase().includes(filterValue.toLowerCase()))
```

Comment:
shouldn't this be equal? what if I want `code-interpreter`, but i will get all "flavors", `code-interpreter-test`, `code-interpreter-v2`, etc

Reply:
you would have to use the templateID as `code-interpreter` is a strict subset of `code-interpreter-test` or `code-interpreter-dev`